src/utils/savedAccounts.js

```diff
@@ -11,36 +11,47 @@ const isValidSavedAccount = ({ publicKey, network, peerAddress }) => {
   }
 };
 
+const fixBackwardsCompatibility = (account) => {
+  // 'address' property was renamed to 'peerAddress' to avoid confusion with account address
+  if (account && account.address) {
```

Comment:
wouldn't it be enough to just check for address property?

```
if (account.address) {
```

Reply:
I'm trying to be defensive because `fixBackwardsCompatibility` is called before `isValidSavedAccount`